Microsoft.Toolkit.Uwp.UI/ImageCache/ImageCache.cs

```diff
@@ -42,10 +42,19 @@ protected class MemoryCacheItem
             public BitmapImage Image { get; set; }
         }
 
+        private class ConcurrentTask
+        {
+            public Task<BitmapImage> Task { get; set; }
+
+            public bool IsAssureOnly { get; set; }
```

Comment:
Not sure I'm convinced by this property name as it gives no indication what it's actually used for. What's the context for this property?